clients/venice-thin-client/src/test/java/com/linkedin/venice/client/schema/RouterBackedSchemaReaderTest.java

```diff
@@ -161,14 +161,14 @@ public void testGetValueSchemaWhenNotExists()
       Mockito.verify(mockClient, Mockito.timeout(TIMEOUT).times(2)).getRaw(Mockito.anyString());
       Schema cachedSchema = schemaReader.getValueSchema(3);
       Assert.assertNull(cachedSchema);
-      Mockito.verify(mockClient, Mockito.timeout(TIMEOUT).times(2)).getRaw(Mockito.anyString());
+      Mockito.verify(mockClient, Mockito.timeout(TIMEOUT).times(3)).getRaw(Mockito.anyString());
```

Comment:
wait but why this adds one more count for each of this call?

Reply:
For missing schema, now it does cache the null, so it will refresh again.

Reply:
Oh I see. Thanks!